docs/included.md

```diff
@@ -13,6 +13,18 @@
   - [API](https://scores.readthedocs.io/en/latest/api.html#scores.continuous.additive_bias)
   - [Tutorial](https://scores.readthedocs.io/en/latest/tutorials/Additive_and_multiplicative_bias.html)
   - [Mean Error (WWRP/WGNE Joint Working Group on Forecast Verification Research)](https://www.cawcr.gov.au/projects/verification/#meanerror)
+* - Consistent Expectile Score
+  - [API](api.md#scores.continuous.consistent_expectile_score)
+  - [Tutorial](project:./tutorials/Consistent_Scores.html)
```

Comment:
```suggestion
  - [Tutorial](project:./tutorials/Consistent_Scores.md)
```
At the end of tutorial links, the html also needs to change to ".md"

Reply:
Ah good pickup - updated now